spring-core/src/test/java/org/springframework/util/AssertTests.java

```diff
@@ -401,6 +402,57 @@ void noNullElementsWithNullElementsAndNullMessageSupplier() {
 			.withMessage(null);
 	}
 
+	@Test
+	void noNullElementsWithCollection() {
+		Assert.noNullElements(asList("foo", "bar"), "enigma");
```

Comment:
For all assertions that are expected not to throw an exception, please rework them to use the following AssertJ construct, where `{}` is the code in question.

```java
assertThatCode(() -> {}).doesNotThrowAnyException();
```